### ISSUE TITLE
DiffBox React version

### ISSUE THREAD

**Issue by SachaG** [meteor/tutorial-tools#9]
We're thinking about using the tutorial tools for the next version of Discover Meteor. This would mean converting DiffBox to React. Is this something you're planning to do as well by any chance?

**Comment by stubailo** [comment]
No I wasn't planning on it but sounds like a good idea! We could use the react version in the official tutorial too probably. @urigo what do you think? Would it be better to switch to react? You're using it on the angular site as well so I wonder what you would prefer.

**Comment by Urigo** [comment]
We are using Blaze on the angular-meteor site right now, but would love to switch...
also I've started thinking more about our meteor.com infrastructure, maybe we should think about the future of that as well?

**Comment by stubailo** [comment]
I don't think it is a problem to use Blaze and React in the same site - Blaze is like 10kb so including both is really not a big deal.

**Comment by Urigo** [comment]
true. so I +1 from me!

**Comment by stubailo** [comment]
Ok Sacha let me know if you start working on this and if I can help!

**Comment by SachaG** [comment]
OK, I'll let you know :)
